index.html

```diff
@@ -0,0 +1,186 @@
+            <span class="material-icons rate checked">star_outline</span>
+            <span class="material-icons rate checked">star_outline</span>
+            <span class="material-icons rate">star_outline</span>
+          </div>
+          <h3 class="card-header">Footwear</h3>
+          <li>Sneakers</li>
+          <li class="price">K300</li>
+        </ul>
+    </div>
+    <div class="card">
+      <img src="img/wolf.JPG" alt="One Blood T-Shirt & Sneaker" class="project-img">
+        <ul>
+          <div class="rating-container">
+            <span class="material-icons rate checked">star_outline</span>
+            <span class="material-icons rate checked">star_outline</span>
+            <span class="material-icons rate checked">star_outline</span>
+            <span class="material-icons rate">star_outline</span>
+            <span class="material-icons rate">star_outline</span>
+          </div>
+          <h3 class="card-header">Footwear</h3>
+          <li>Sneakers</li>
+          <li class="price">K300</li>
+        </ul>
+    </div>
+</div>
+<div class="best-seller-section-container">
+  <div class="best-seller-section">
+    <h2>Trending Now</h2>
+    <span class="see-wrapper">
+      <p>See all </p>
+      <span class="material-icons">chevron_right</span>
+    </span>
+  </div> 
+</div>
+<div class="trending-card-container">
+  <div class="trending-card">
+    <img src="img/trending/L1.JPG" class="project-img" alt="">
+  </div>
+  <div class="trending-card">
+    <img src="img/trending/L2.JPG" class="project-img" alt="">
+  </div>
+  <div class="trending-card">
+    <img src="img/trending/L3.JPG" class="project-img" alt="">
+  </div>
+  <div class="trending-card">
+    <img src="img/trending/L4.JPG" class="project-img" alt="">
+  </div>
+  <div class="trending-card">
+    <img src="img/trending/M1.jpg" class="project-img" alt="">
+  </div>
+  <div class="trending-card">
+    <img src="img/trending/M2.JPG" class="project-img" alt="">
+  </div>
+</div>
+<footer class="footer">
+  <a href="https://www.instagram.com/eastern_designs/"><i class="fab fa-instagram"></i></a>
+  <a href="https://web.facebook.com/Eastern-Designs-2169913273129712/"><i class="fab fa-facebook"></i></a>
+  <a href="#"><i class="fab fa-twitter"></i></a>
+  <a href="#"><i class="fab fa-whatsapp"></i></a>
+</footer>
+<script src="app.js"></script>
+</body>
+</html>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+<!-- <header>
+  <nav>
+    <div class="hamburger">
+      <span class="bar"></span>
+      <span class="bar"></span>
+      <span class="bar"></span>
+  </div>
+  </nav> -->
+  <!-- <nav class="navbar">
+    <span class="material-icons" id="burger">menu</span>
+    <span>Eastern Designs</span>
+    <span class="material-icons">search</span>
+  </nav> -->
+  <!-- <div class="nav-links">
+    <a href="#news">News</a>
+    <a href="#contact">Contact</a>
+    <a href="#about">About</a>
+  </div> -->
+<!-- </header> -->
```

Comment:
- [ ] Kindly remove the commented code.It's not a good practice to have commented code in your file.